File: appService/IndoorService/application/models/Edifice.php
<?php

/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

/**
 * Description of Edifice
 *
 * @author Luis Gumucio
 */
class Edifice extends CI_Model {

    function __construct() {
        //call construct this model
        parent::__construct();
    }

    /**
     * metodo para insertar a la base datos
     */
    function insertEdifice($id, $edifice,$photo, $idFloor, $nameFloor, $numberFloor) {
        $this->db->set('idEdifice', $id);
        $this->db->set('NameEdifice', $edifice);
        $this->db->set('photo', $photo);
        $this->db->insert('edifice');

        $this->db->set('idFloor', $idFloor);
        $this->db->set('NameFloor', $nameFloor);
        $this->db->set('NumberFloor', $numberFloor);
        $this->db->set('Edifice_idEdifice', $id);
        $this->db->insert('floor');
        return $id;
    }

    /**
     * metodo para obtener todos los edificios de la base de datos
     */
    function getEdifice() {
        $edifice['edifice'] = array();
        $query = $this->db->get('edifice');
        $result = $query->result_array();
        foreach ($query->result_array() as $row) {
            array_push($edifice['edifice'], array(
                'idEdifice' => $row['idEdifice'],
                'NameEdifice' => $row['NameEdifice'],
                'photo' => base64_encode($row['photo'])
            ));
        }
        return $edifice;
    }

}

File: src/org/entity/Edifice.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package org.entity;

/**
 *
 * @author Luis Gumucio
 */
public class Edifice {

    private String idEdifice;
    private String nameEdifice;

    public Edifice(String idEdifice, String nameEdifice) {
        this.idEdifice = idEdifice;
        this.nameEdifice = nameEdifice;
    }

    public String getIdEdifice() {
        return idEdifice;
    }

    public void setIdEdifice(String idEdifice) {
        this.idEdifice = idEdifice;
    }

    public String getNameEdifice() {
        return nameEdifice;
    }

    public void setNameEdifice(String nameEdifice) {
        this.nameEdifice = nameEdifice;
    }

    @Override
    public int hashCode() {
        int hash = 7;
        hash = 67 * hash + (this.idEdifice != null ? this.idEdifice.hashCode() : 0);
        return hash;
    }

    @Override
    public boolean equals(Object obj) {
        if (obj == null) {
            return false;
        }
        if (getClass() != obj.getClass()) {
            return false;
        }
        final Edifice other = (Edifice) obj;
        if ((this.idEdifice == null) ? (other.idEdifice != null) : !this.idEdifice.equals(other.idEdifice)) {
            return false;
        }
        return true;
    }
}
